### PR TITLE
Add doc prop

### PR DESCRIPTION
**Pull request by alexisseppolabat**
fixes: #94

### CODE REVIEW

File: packages/core/src/lib/visit/visit.ts [hunk 1]
```diff
@@ -79,6 +79,35 @@ export const visit: Visiter = ({ deps, node, metadata }): ts.Node => {
       ],
       deps: deps,
     });
+
+    // Append the JSDoc comments from the input node onto the output node
+    if (Object.keys(node).includes("jsDoc")) {
+      const getSrcFile: (node: ts.Node) => ts.SourceFile = (node) => {
```
Comment (Zachiah) [inline packages/core/src/lib/visit/visit.ts:85]: Can we please put this in the utils? This is just a pure function right? Ie. it's not a closure?

File: .changeset/yellow-spies-design.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@ts-r/core": minor
```
Comment (Zachiah) [inline .changeset/yellow-spies-design.md:2]: Can we change this to be patch? I'd like to keep the version numbers down

**Review by Zachiah — CHANGES_REQUESTED** [review]

File: .changeset/yellow-spies-design.md (hunk 2 above)
Comment (alexisseppolabat) [reply .changeset/yellow-spies-design.md:2]: Sure

File: packages/core/src/lib/visit/visit.ts (hunk 1 above)
Comment (alexisseppolabat) [reply packages/core/src/lib/visit/visit.ts:85]: Sure

**Review by Zachiah — APPROVED** [review]

**Pull request merged**